src/modules/vtol_att_control/standard.cpp

```diff
@@ -368,6 +368,12 @@ void Standard::fill_actuator_outputs()
 
 	_actuators_out_1->control[actuator_controls_s::INDEX_THROTTLE] = _pusher_throttle;
 
+	/* translate MC actuators to FW actuators */
+	if (_vtol_schedule.flight_mode == TRANSITION_TO_FW || _vtol_schedule.flight_mode == TRANSITION_TO_MC) {
+		_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = -_actuators_mc_in->control[actuator_controls_s::INDEX_YAW];
```

Comment:
This can't be good. If the controller commands yaw but instead it starts to roll.. The positioning is already solved in the blending branch and not the main issue here.